src/networking.h

```diff
@@ -23,6 +23,12 @@
 
 #include "netlink.h"
 
+/* openvswitch will place the vhost-user socket in a particular location,
+ * and we chose to name the interface ovd_<ip address> during endpoint creation
+ * in the OVSDPDK Docker plugin.  OVS_PORT_PATH is used to identify this file.
+ */
+#define OVS_PORT_PATH "/tmp/v_%s"
```

Comment:
ah ok. So unified handling for both dpdk and VPP. Then what is the role of the /tmp/v_file

"The runtime determines that a vhost-user enbaled interface is desired if
the IP for a discovered network interface matches the address appended to a
/tmp/v_ file."

Also there are a couple of typos in the commit message